ketcher-autotests/tests/Indigo-Tools/Layout/layout.spec.ts

```diff
@@ -1,14 +1,25 @@
 import { Page, test } from '@playwright/test';
+import {
+  selectPartOfChain,
```

Comment:
I propose to move this methods to utils instead of importing them from other tests